lib/class-core.php

```diff
@@ -25,17 +25,21 @@ class Core {
 	 */
 	public function setup() {
 		add_action( 'template_redirect', [ $this, 'redirects' ] );
+		add_action( 'after_setup_theme', [ $this, 'alias_redirects' ] );
 
 		// Filter permalinks.
-		add_filter( 'post_link',         [ $this, 'post_link' ],     10, 2 );
-		add_filter( 'page_link',         [ $this, 'post_link' ],     10, 2 );
-		add_filter( 'attachment_link',   [ $this, 'post_link' ],     10, 2 );
-		add_filter( 'post_type_link',    [ $this, 'post_link' ],     10, 2 );
+		add_filter( 'post_link', [ $this, 'post_link' ],     10, 2 );
```

Comment:
If you are doing cleanup then you might want to remove the spaces before the `10,2` here as well.  Same goes for the below lines of code.